package.json

```diff
@@ -85,7 +86,8 @@
     "serve:package": "webpack-dev-server --config webpack/webpack.config.js --hot --inline --history-api-fallback",
     "samples:test": "NODE_ENV=test wdio",
     "tooling": "babel-node ./tooling/index.js",
-    "postinstall": "patch-package"
+    "postinstall": "patch-package",
+    "version": "./tooling/version.sh"
```

Comment:
According to the docs for [npm version](https://docs.npmjs.com/cli/version), step 3 NPM bumps the version and then executes this script.
Does this not conflict with the script in the CI that already bumps the version?

Reply:
it shouldn't since we're explicitly setting it in the circleci config
`npm version ${VERSION} -m 'chore(release): v%s [skip ci]'`

Reply:
Step 3 says:
```
Bump version in package.json as requested
```

So I'm assuming the versioning will work is:
1. Current version [bump CI step](https://github.com/webex/webex-js-sdk/blob/136974e8dc6d45a168dec2a46c2a39a5c0e8c5d3/.circleci/config.yml#L329) bumps version in all `package.json` packages but the [top-level `package.json`](https://github.com/webex/webex-js-sdk/blob/master/package.json)
2. `npm version` will bump top-level [top-level `package.json`](https://github.com/webex/webex-js-sdk/blob/master/package.json)
3. `version` script will get executed (e.g. building + changelog generation)